backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -1,7 +1,6 @@
 package com.crm.model.db;
 
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Tutaj to samo

Reply:
Tutaj również jest ok, tak samo jak wyżej

Reply:
Nie jest. Nie importujmy całego pakietu lombok do klasy. Nie jest to potrzebne. Importujmy tylko to czego używamy.

Reply:
Poprawione